Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1

```
@@ -607,7 +607,8 @@ function InvokeDTAExecHostExe([string] $Version, [System.Management.Automation.P
     $exePath = Join-Path -Path $vsRoot -ChildPath $ExeName
     $exePath = "'" + $exePath + "'"
 
-    $session = New-PSSession -ComputerName . -Credential $MachineCredential
+    $session = CreateNewSession
+
     Invoke-Command -Session $session -ErrorAction Continue -ErrorVariable err -OutVariable out -scriptBlock { schtasks.exe /create /TN:DTAConfig /TR:$args /F /RL:HIGHEST /SD:01/01/2050 /SC:ONCE /ST:00:00 ; schtasks.exe /run /TN:DTAConfig } -ArgumentList $exePath
```

can we place this code in a try block and catch all exceptions ?